test/Microsoft.ML.Core.Tests/UnitTests/TestVBuffer.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Xunit;
+
+namespace Microsoft.ML.Core.Tests.UnitTests
+{
+    public sealed class TestVBuffer
```

Comment:
(Nit) I would name this class and file “TestVBufferUtils” instead, because that is what it is testing.